Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/inbound/deferred/request/StandardInboundDocSubmissionDeferredRequestTest.java

```diff
@@ -170,38 +165,40 @@ public void badIncomingAssertion() {
         AssertionType assertion = new AssertionType();
         XDRAcknowledgementType expectedResponse = new XDRAcknowledgementType();
 
-        AdapterDocSubmissionDeferredRequestProxyObjectFactory adapterFactory = mock(AdapterDocSubmissionDeferredRequestProxyObjectFactory.class);
+        AdapterDocSubmissionDeferredRequestProxyObjectFactory adapterFactory
+            = mock(AdapterDocSubmissionDeferredRequestProxyObjectFactory.class);
         PropertyAccessor propertyAccessor = mock(PropertyAccessor.class);
         XDRPolicyChecker policyChecker = mock(XDRPolicyChecker.class);
-        XDRAuditLogger auditLogger = mock(XDRAuditLogger.class);
-        AdapterDocSubmissionDeferredRequestErrorProxyObjectFactory errorAdapterFactory = mock(AdapterDocSubmissionDeferredRequestErrorProxyObjectFactory.class);
-        AdapterDocSubmissionDeferredRequestErrorProxy errorAdapter = mock(AdapterDocSubmissionDeferredRequestErrorProxy.class);
-
+        DocSubmissionDeferredRequestAuditLogger auditLogger = mock(DocSubmissionDeferredRequestAuditLogger.class);
+        AdapterDocSubmissionDeferredRequestErrorProxyObjectFactory errorAdapterFactory
```

Comment:
Same comment as creating objects in method and calling it before the actual test methods are called?